paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -58,12 +48,21 @@ export class OneKeyLib implements IOneKeyLib {
   private clientHostname: string;
   triggerRedirectIfNeeded: boolean;
   private thirdPartyCookiesSupported: boolean | undefined;
+  private auditLogStorageService: IAuditLogStorageService;
```

Comment:
FAR: we should move these declarations into the constructor to keep code concise